eslint-plugin/README.md

```diff
@@ -67,6 +67,7 @@ to have more information about the integration.
 
 | Name                                                                           | Description                                   | ⚠️ |
 | :----------------------------------------------------------------------------- | :-------------------------------------------- | :- |
+| [no-import-all-from-library](docs/rules/no-import-all-from-library.md)         | Should not import all from library            | ✅  |
```

Comment:
to ben homogeneous with other repositories, why not write these rule descriptions in a `RULES.md` root file ?

Reply:
It's a common practice for an ESLint plugin to list rules directly into README file, in order to display them on the [npmjs.com package page](https://www.npmjs.com/package/@ecocode/eslint-plugin?activeTab=readme) (this is the only file used for display). When we have a common repository with all the rules documentation, it will be even better